content/blog/2020-08-27-august-20-community-gems.md

```diff
@@ -0,0 +1,195 @@
+---
+title: August '20 Community Gems
+date: 2020-08-27
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML
+  adds pictures to your GitHub and GitLab comments.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML
+  adds pictures to your GitHub and GitLab comments.
+picture: 2020-08-27/Gems_Aug_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/august-20-community-gems/477
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Hyperparameters
+  - Git LFS
+  - Pipeline
+---
+
+Here are some of our top Q&A's from around the community. With the launch of
+[CML](https://cml.dev) earlier in the month, we've got some new ground to cover!
+
+## DVC questions
+
+### [Q: What's the relationship between your DVC remote and cache? If I have an external cache, do I really need a DVC remote?](https://discordapp.com/channels/485586884165107732/563406153334128681/747588572479094866)
+
+You can think of your DVC remote similar to your Git remote, but for data and
+model artifacts- it's a place to backup and share artifacts. It also gives you
+methods to push and pull those artifacts to and from your team.
+
+Your DVC cache (by default, it's located in `.dvc/cache`) serves a similar
+purpose to your Git objects database (which is by default located in
+`.git/objects`). They're both _local_ caches that store files (including various
+versions of them) in a content-addressable format, which helps you quickly
+checkout different versions to your local workspace. The difference is that
+`.dvc/cache` is for data/model artifacts, and `.git/objects` is for code.
+
+Usually, your DVC remote is a superset of `.dvc/cache`- everything in your cache
+is a copy of something in your remote (though there may be files in your DVC
+remote that are not in your cache (and vice versa) if you have never attempted
+to `push` or `pull` them locally).
+
+In theory, if you are using an
+[external cache](https://dvc.org/doc/command-reference/destroy#example-external-cache-directory)-
```

Comment:
destroy link feels a bit strange

let's change the link to this one - https://dvc.org/doc/use-cases/shared-development-server ... we can say that this is one of the useful scenarios for the shared cache